docs/make.bat

```diff
@@ -46,15 +45,14 @@ if "%1" == "clean" (
 	for /d %%i in (%BUILDDIR%\*) do rmdir /q /s %%i
 	del /q /s %BUILDDIR%\*
 	for /d %%i in (source\*) do rmdir /q /s %%i
-
+	del /q /s source\extensions.rst
```

Comment:
Do you think the name matters ?
It is just a filename that is being generated and contains the list of notebook extensions.